docs/zh/docs/advanced/settings.md

```diff
@@ -0,0 +1,439 @@
+# 设置和环境变量
+
+在许多情况下，您的应用程序可能需要一些外部设置或配置，例如密钥、数据库凭据、电子邮件服务的凭据等等。
+
+这些设置中的大多数是可变的（可以更改的），比如数据库的 URL。而且许多设置可能是敏感的，比如密钥。
+
+因此，通常会将它们提供为由应用程序读取的环境变量。
+
+## 环境变量
+
+!!! tip
+    如果您已经知道什么是"环境变量"以及如何使用它们，请随意跳到下面的下一节。
+
+环境变量（也称为"env var"）是一种存在于 Python 代码之外、存在于操作系统中的变量，可以被您的 Python 代码（或其他程序）读取。
+
+您可以在 shell 中创建和使用环境变量，而无需使用 Python：
+
+=== "Linux、macOS、Windows Bash"
+
+    <div class="termy">
+
+    ```console
+    // 您可以创建一个名为 MY_NAME 的环境变量
+    $ export MY_NAME="Wade Wilson"
+
+    // 然后您可以与其他程序一起使用它，例如
+    $ echo "Hello $MY_NAME"
+
+    Hello Wade Wilson
+    ```
+
+    </div>
+
+=== "Windows PowerShell"
+
+    <div class="termy">
+
+    ```console
+    // 创建一个名为 MY_NAME 的环境变量
+    $ $Env:MY_NAME = "Wade Wilson"
+
+    // 与其他程序一起使用它，例如
+    $ echo "Hello $Env:MY_NAME"
+
+    Hello Wade Wilson
+    ```
+
+    </div>
+
+### 在 Python 中读取环境变量
+
+您还可以在 Python 之外的地方（例如终端中或使用任何其他方法）创建环境变量，然后在 Python 中读取它们。
+
+例如，您可以有一个名为 `main.py` 的文件，其中包含以下内容：
+
+```Python hl_lines="3"
+import os
+
+name = os.getenv("MY_NAME", "World")
+print(f"Hello {name} from Python")
+```
+
+!!! tip
+    `os.getenv()` 的第二个参数是要返回的默认值。
```

Comment:
```md
`os.getenv()` 
```
->
```  md
<a href="https://docs.python.org/3.8/library/os.html#os.getenv" class="external-link" target="_blank">`os.getenv()`</a>
```

Reply:
Thanks,The modification has been completed.